common/model-views.cpp

```diff
@@ -2166,89 +2205,267 @@ namespace rs2
+
+                ImGui::SameLine();
+
+                std::string res;
+                if (profile.as<rs2::video_stream_profile>())
+                    res = to_string() << size.x << "x" << size.y << ",  ";
+                label = to_string() << res << truncate_string(rs2_format_to_string(profile.format()), 9) << ", ";
+                ImGui::Text("%s", label.c_str());
+                if (ImGui::IsItemHovered())
                 {
-                     ImGui::SetTooltip("Timestamp: Global Time");
+                    ImGui::SetTooltip("%s", "Stream Resolution, Format");
                 }
-                else
+
+                ImGui::SameLine();
+
+                label = to_string() << "FPS: " << std::setprecision(2) << std::setw(7) << std::fixed << fps.get_fps();
+                ImGui::Text("%s", label.c_str());
+                if (ImGui::IsItemHovered())
                 {
-                    ImGui::SetTooltip("Timestamp: Hardware Clock");
+                    ImGui::SetTooltip("%s", "FPS is calculated based on timestamps and not viewer time");
                 }
+
+                line_y += ImGui::GetTextLineHeight() + 5;
             }
+        }
 
-            ImGui::SameLine();
+        
 
-            label = to_string() << " Frame: " << std::left <<  frame_number;
-            ImGui::Text("%s", label.c_str());
+        if (show_metadata)
+        {
+            std::vector<attribute> stream_details;
+
+            if (true) // Always show stream details options
+            {
+                stream_details.push_back({ "Frame Timestamp",
+                    to_string() << std::fixed << std::setprecision(1) << timestamp, 
+                    "Frame Timestamp is normalized represetation of when the frame was taken.\n"
+                    "It's a property of every frame, so when exact creation time is not provided by the hardware, an approximation will be used.\n"
+                    "Clock Domain feilds helps interpret the meaning of timestamp\n"
+                    "Timestamp is measured in milliseconds, and is allowed to roll-over (reset to zero) in some situations" });
+                stream_details.push_back({ "Clock Domain",
+                    to_string() << rs2_timestamp_domain_to_string(timestamp_domain), 
+                    "Clock Domain describes the format of Timestamp feild. It can be one of the following:\n"
+                    "1. System Time - When no hardware timestamp is available, system time of arrival will be used.\n"
+                    "                 System time benefits from being comparable between device, but suffers from not being able to approximate latency.\n"
+                    "2. Hardware Clock - Hardware timestamp is attached to the frame by the device, and is consistent accross device sensors.\n"
+                    "                    Hardware timestamp encodes percisely when frame was captured, but cannot be compared across devices\n"
```

Comment:
precisely

Reply:
~~cannot be compared across devices~~ cannot be used for inter-device synchronization.